src/app/zap-templates/zcl/data-model/all.xml

```diff
@@ -53,6 +53,7 @@
    <xi:include href="chip/pump-configuration-and-control-cluster.xml" />
    <xi:include href="chip/pwm-cluster.xml" />
    <xi:include href="chip/relative-humidity-measurement-cluster.xml" />
+   <xi:include href="chip/replacable-monitoring-cluster.xml.xml" />
```

Comment:
`xml.xml` seems like a duplicated extension. Is this file used in any way?

Reply:
Oh yeah! Thanks for spotting! I actually have no idea why this file should be in this list tbh. I followed a guide which said that the cluster definition files need to be added in this file.